packs/twitter/actions/direct_message.yaml

```diff
@@ -0,0 +1,15 @@
+---
+  name: "direct_message"
+  runner_type: "run-python"
+  description: "Direct message a user."
+  enabled: true
+  entry_point: "direct_message.py"
+  parameters:
+    user:
```

Comment:
Probably want to not use runner.  Since it is a runner parameter for other runners (local, remote), it could be confusing to content writers/users.